docs/deployment.md

```diff
@@ -2,7 +2,8 @@
 
 Deployments of Basemaps are managed with github actions.
 
-To trigger a deployment, make sure your branch is up to date and run the version bump script [../scripts/version.bump.sh](../scripts/version.bump.sh)
+To trigger a deployment, make sure your branch is up to date and run the version bump script [version.bump.sh](https://github.com/linz/basemaps/blob/master/scripts/version.bump.sh).
```

Comment:
Why we not using relative path for local scripts?

Reply:
They don't work, https://dev.basemaps.linz.govt.nz/docs/deployment/